umbraco/Umbraco-CMS
# v10: Fix build warnings in Web.Common

# Notes
- Removed lots of warnings again, i think the interesting stuff is all the redundant "?" checks, cause even though the compiler will give a warning for some of them, its not all that are actually redudant (see the fix test commit, some where failing cause i had removed the ? checks)

- Like in https://github.com/umbraco/Umbraco-CMS/pull/12332 please be aware of the [Obsolete] attrubute, dotnet format adds these when using obsolete methods -.-

# How to test
- Be on v10/dev
- Open your .`globalconfig` file
- On line 44-49 change the severity now to warning instead of suggestion
- Build the project and note the number of build warnings
- Switch branch to this one and build again
- Build warnings should be reduced 🙏
